aspnetcore/blazor/webassembly-lazy-load-assemblies.md

```diff
@@ -41,6 +41,14 @@ Only assemblies that are used by the app can be lazily loaded. The linker strips
 > </ItemGroup>
 > ```
 
+## Prerendering
+
+When using lazy loading with prerendering in a hosted Blazor solution, manually register `LazyAssemblyLoader` in the *Server* project's `Startup.ConfigureServices` method (`Startup.cs`):
+
+```csharp
+services.AddScoped<LazyAssemblyLoader>();
```

Comment:
```suggestion
services.AddSingleton<LazyAssemblyLoader>();
```

Reply:
What is package (dll) to get the LazyAssemblyLoader?

Reply:
It's in `Microsoft.AspNetCore.Components.WebAssembly` ...

https://github.com/dotnet/aspnetcore/blob/master/src/Components/WebAssembly/WebAssembly/src/Services/LazyAssemblyLoader.cs

It's best to ask support questions on a support channel, as we only handle doc issues on this repo.

* [Stack Overflow](https://stackoverflow.com/questions/tagged/blazor)
* [ASP.NET Core Slack](http://tattoocoder.com/aspnet-slack-sign-up/)
* [ASP.NET Gitter](https://gitter.im/aspnet/Blazor)

Reply:
btw ... the 5.0 preview doc on it is available at :point_right: https://docs.microsoft.com/en-us/aspnet/core/blazor/webassembly-lazy-load-assemblies?view=aspnetcore-5.0